### PR TITLE
Update anvil-rstudio-bioconductor image

### PR DESCRIPTION
**Pull request by nturaga**
This will update the anvil-rstudio-bioconductor docker image that was rebuilt.

- this will add additional sys dependencies to make more Bioconductor packages work.

Related PR https://github.com/anvilproject/anvil-docker/pull/41

### CODE REVIEW

**Comment by Qi77Qi** [comment]
the change itself lgtm, but if we merge this PR, we should re-generate custom VM images soon. It's probably the easiest if we wait until next monolith release before merging this PR cuz we have some other changes going on that updates UI versions too

**Comment by nturaga** [comment]
> 

Can you elaborate on this a little bit more? I was under the assumption this was the lowest rung on the ladder for this rstudio bioconductor image. What do you mean regenerate VM images?

**Review by Qi77Qi — APPROVED** [review]

**Pull request merged**
